contents/handbook/growth/marketing/ownership.md

```diff
@@ -19,34 +19,25 @@ showTitle: true
 | **SEO landing pages**           | Andy                 |
 | **Social media**                | Andy                 |
 | **Product comparisons**         | Andy                 |
+| **Video**         | Alex                 |
 | **Where people hear about us**  | Andy                 |
 
-## Owned by Comms
+## Owned by Brand & Vibes
 
 | &nbsp                                   | **Individual owner** |
 |-----------------------------------------|----------------------|
 | **Case studies**                        | Joe                  |
 | **Changelog**                           | Joe                  |
 | **Emailing customers**                  | Joe                  |
 | **Events**                              | Daniel               |
-| **Growth review**                       | Brian                |
```

Comment:
I realize there isn't really anyone do to this without @BrianAtPosthog, but does this mean we are getting rid of the marketing growth review?